diff -up blender-2.48a/config/linux2-config.py.org blender-2.48a/config/linux2-config.py
--- blender-2.48a/config/linux2-config.py.org	2008-12-27 13:17:57.000000000 +0100
+++ blender-2.48a/config/linux2-config.py	2008-12-27 13:17:59.000000000 +0100
@@ -179,6 +179,12 @@ CCFLAGS = ['-pipe','-fPIC','-funsigned-c
 
 CPPFLAGS = ['-DXP_UNIX']
 CXXFLAGS = ['-pipe','-fPIC','-funsigned-char','-fno-strict-aliasing']
+
+import os
+RPM_OPT_FLAGS = os.environ['RPM_OPT_FLAGS'].split()
+CCFLAGS += RPM_OPT_FLAGS
+CXXFLAGS += RPM_OPT_FLAGS
+
 REL_CFLAGS = ['-O2']
 REL_CCFLAGS = ['-O2']
 ##BF_DEPEND = True
diff -up blender-2.48a/extern/SConscript.org blender-2.48a/extern/SConscript
--- blender-2.48a/extern/SConscript.org	2008-10-22 18:38:21.000000000 +0200
+++ blender-2.48a/extern/SConscript	2008-12-27 13:16:31.000000000 +0100
@@ -2,8 +2,6 @@
 
 Import('env')
 
-SConscript(['glew/SConscript'])
-
 if env['WITH_BF_GAMEENGINE']:
     SConscript(['qhull/SConscript',
             'solid/SConscript'])
diff -up blender-2.48a/SConstruct.org blender-2.48a/SConstruct
--- blender-2.48a/SConstruct.org	2008-10-22 18:38:43.000000000 +0200
+++ blender-2.48a/SConstruct	2008-12-27 13:16:31.000000000 +0100
@@ -217,7 +217,7 @@ if env['OURPLATFORM'] == 'linux2' :
 
         def CheckFreeAlut(context,env):
             context.Message( B.bc.OKGREEN + "Linux platform detected:\n  checking for FreeAlut... " + B.bc.ENDC )
-            env['LIBS'] = 'alut'
+            env['LIBS'] = ['alut', 'openal']
             result = context.TryLink(mylib_test_source_file, '.c')
             context.Result(result)
             return result
