diff -up blender-2.68a/source/blender/blenfont/intern/blf_lang.c.syspath blender-2.68a/source/blender/blenfont/intern/blf_lang.c
--- blender-2.68a/source/blender/blenfont/intern/blf_lang.c.syspath	2013-06-21 14:33:19.000000000 +0200
+++ blender-2.68a/source/blender/blenfont/intern/blf_lang.c	2013-09-01 15:58:24.422656833 +0200
@@ -186,7 +186,7 @@ EnumPropertyItem *BLF_RNA_lang_enum_prop
 void BLF_lang_init(void)
 {
 #ifdef WITH_INTERNATIONAL
-	const char * const messagepath = BLI_get_folder(BLENDER_DATAFILES, "locale");
+  const char * const messagepath = "/usr/share/locale";
 
 	if (messagepath) {
 		bl_locale_init(messagepath, TEXT_DOMAIN_NAME);
