diff -up blender-2.68a/source/blender/blenfont/intern/blf_translation.c.droid blender-2.68a/source/blender/blenfont/intern/blf_translation.c
--- blender-2.68a/source/blender/blenfont/intern/blf_translation.c.droid	2013-03-20 19:42:09.000000000 +0100
+++ blender-2.68a/source/blender/blenfont/intern/blf_translation.c	2013-09-04 17:05:04.775210317 +0200
@@ -59,7 +59,7 @@ unsigned char *BLF_get_unifont(int *unif
 {
 #ifdef WITH_INTERNATIONAL
 	if (unifont_ttf == NULL) {
-		const char * const fontpath = BLI_get_folder(BLENDER_DATAFILES, "fonts");
+		const char * const fontpath = "/usr/share/fonts/blender";
 		if (fontpath) {
 			char unifont_path[1024];
 
@@ -94,7 +94,7 @@ unsigned char *BLF_get_unifont_mono(int
 {
 #ifdef WITH_INTERNATIONAL
 	if (unifont_mono_ttf == NULL) {
-		const char *fontpath = BLI_get_folder(BLENDER_DATAFILES, "fonts");
+		const char *fontpath = "/usr/share/fonts/blender";
 		if (fontpath) {
 			char unifont_path[1024];
 
