diff -up blender-2.45/tools/btools.py.scons blender-2.45/tools/btools.py
--- blender-2.45/tools/btools.py.scons	2007-09-18 07:00:48.000000000 +0200
+++ blender-2.45/tools/btools.py	2008-04-27 21:23:27.000000000 +0200
@@ -1,7 +1,10 @@
 import os
 import os.path
 import SCons.Options
-import SCons.Options.BoolOption
+try:
+    import SCons.Options.BoolOption
+except ImportError:
+    pass
 try:
     import subprocess
 except ImportError:
