diff -up blender-2.47/extern/SConscript.org blender-2.47/extern/SConscript
--- blender-2.47/extern/SConscript.org	2008-10-14 22:13:44.000000000 +0200
+++ blender-2.47/extern/SConscript	2008-10-14 22:13:57.000000000 +0200
@@ -2,8 +2,6 @@
 
 Import('env')
 
-SConscript(['glew/SConscript'])
-
 if env['WITH_BF_GAMEENGINE']:
     SConscript(['qhull/SConscript',
             'solid/SConscript'])
diff -up blender-2.47/SConstruct.org blender-2.47/SConstruct
--- blender-2.47/SConstruct.org	2008-05-01 16:22:21.000000000 +0200
+++ blender-2.47/SConstruct	2008-10-14 22:13:27.000000000 +0200
@@ -216,7 +216,7 @@ if env['OURPLATFORM'] == 'linux2' :
 
         def CheckFreeAlut(context,env):
             context.Message( B.bc.OKGREEN + "Linux platform detected:\n  checking for FreeAlut... " + B.bc.ENDC )
-            env['LIBS'] = 'alut'
+            env['LIBS'] = ['alut', 'openal']
             result = context.TryLink(mylib_test_source_file, '.c')
             context.Result(result)
             return result
