diff -up blender-2.61/intern/elbeem/intern/ntl_vector3dim.h.gcc47 blender-2.61/intern/elbeem/intern/ntl_vector3dim.h
--- blender-2.61/intern/elbeem/intern/ntl_vector3dim.h.gcc47	2012-02-08 18:57:14.006686472 +0100
+++ blender-2.61/intern/elbeem/intern/ntl_vector3dim.h	2012-02-08 19:01:36.319268168 +0100
@@ -27,6 +27,12 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+/* absolute value */
+template < class T >
+inline T
+ABS( T a )
+{ return (0 < a) ? a : -a ; }
+
 // hack for MSVC6.0 compiler
 #ifdef _MSC_VER
 #if _MSC_VER < 1300
diff -up blender-2.61/intern/elbeem/intern/utilities.h.gcc47 blender-2.61/intern/elbeem/intern/utilities.h
--- blender-2.61/intern/elbeem/intern/utilities.h.gcc47	2011-12-13 20:42:21.000000000 +0100
+++ blender-2.61/intern/elbeem/intern/utilities.h	2012-02-08 19:01:37.519253177 +0100
@@ -10,8 +10,8 @@
  *
  *****************************************************************************/
 #ifndef UTILITIES_H
-#include "ntl_vector3dim.h"
 
+#include "ntl_vector3dim.h"
 
 /* debugging outputs , debug level 0 (off) to 10 (max) */
 #ifdef ELBEEM_PLUGIN
@@ -177,12 +177,6 @@ inline T
 MAX( T a, T b )
 { return (a < b) ? b : a ; }
 
-/* absolute value */
-template < class T >
-inline T
-ABS( T a )
-{ return (0 < a) ? a : -a ; }
-
 /* sign of the value */
 template < class T >
 inline T
@@ -208,4 +202,5 @@ SQUARE( T a )
 
 
 #define UTILITIES_H
+
 #endif
