diff -up blender-2.45/tools/btools.py.scons blender-2.45/tools/btools.py
--- blender-2.45/tools/btools.py.scons	2007-09-18 07:00:48.000000000 +0200
+++ blender-2.45/tools/btools.py	2008-04-24 19:09:11.000000000 +0200
@@ -1,7 +1,6 @@
 import os
 import os.path
 import SCons.Options
-import SCons.Options.BoolOption
 try:
     import subprocess
 except ImportError:
@@ -12,7 +11,7 @@ import shutil
 import sys
 
 Options = SCons.Options
-BoolOption = SCons.Options.BoolOption
+BoolOption = SCons.Options.BoolOption 
 
 def print_arguments(args, bc):
     if len(args):
