diff -up blender-2.56-beta-source/source/gameengine/GameLogic/SCA_PythonController.cpp.py32 blender-2.56-beta-source/source/gameengine/GameLogic/SCA_PythonController.cpp
--- blender-2.56-beta-source/source/gameengine/GameLogic/SCA_PythonController.cpp.py32	2011-01-16 22:44:18.706809999 +0100
+++ blender-2.56-beta-source/source/gameengine/GameLogic/SCA_PythonController.cpp	2011-01-16 22:44:54.668810004 +0100
@@ -408,7 +408,7 @@ void SCA_PythonController::Trigger(SCA_L
 		 */
 
 		excdict= PyDict_Copy(m_pythondictionary);
-		resultobj = PyEval_EvalCode((PyCodeObject*)m_bytecode, excdict, excdict);
+		resultobj = PyEval_EvalCode((PyObject*)m_bytecode, excdict, excdict);
 		/* PyRun_SimpleString(m_scriptText.Ptr()); */
 		break;
 	}
