diff -Naur blender-cycles-2.58.orig/source/blender/blenkernel/CMakeLists.txt blender-cycles-2.58/source/blender/blenkernel/CMakeLists.txt
--- blender-cycles-2.58.orig/source/blender/blenkernel/CMakeLists.txt	2011-06-28 10:00:42.821785000 -0500
+++ blender-cycles-2.58/source/blender/blenkernel/CMakeLists.txt	2011-07-07 10:41:51.119709118 -0500
@@ -144,7 +144,6 @@
 	intern/unit.c
 	intern/world.c
 	intern/writeavi.c
-	intern/writeffmpeg.c
 	intern/writeframeserver.c
 	
 	BKE_DerivedMesh.h
@@ -225,7 +224,6 @@
 	BKE_utildefines.h
 	BKE_world.h
 	BKE_writeavi.h
-	BKE_writeffmpeg.h
 	BKE_writeframeserver.h
 	depsgraph_private.h
 	intern/CCGSubSurf.h
diff -Naur blender-cycles-2.58.orig/source/gameengine/VideoTexture/CMakeLists.txt blender-cycles-2.58/source/gameengine/VideoTexture/CMakeLists.txt
--- blender-cycles-2.58.orig/source/gameengine/VideoTexture/CMakeLists.txt	2011-06-21 10:47:18.042624000 -0500
+++ blender-cycles-2.58/source/gameengine/VideoTexture/CMakeLists.txt	2011-07-07 10:43:00.184382880 -0500
@@ -67,7 +67,6 @@
 	PyTypeList.cpp
 	Texture.cpp
 	VideoBase.cpp
-	VideoFFmpeg.cpp
 	blendVideoTex.cpp
 
 	BlendType.h
@@ -86,7 +85,6 @@
 	PyTypeList.h
 	Texture.h
 	VideoBase.h
-	VideoFFmpeg.h
 )
 
 if(WITH_CODEC_FFMPEG)
