--- blender2.42/SConstruct.org	2006-06-23 20:52:31.000000000 +0200
+++ blender2.42/SConstruct	2006-07-17 19:09:52.000000000 +0200
@@ -166,6 +166,8 @@
 
 printdebug = B.arguments.get('BF_LISTDEBUG', 0)
 
+env['CCFLAGS'] = env['CCFLAGS'].split()
+
 # see if this linux distro has libalut
 
 if env['OURPLATFORM'] == 'linux2' :
@@ -181,7 +183,7 @@
 
         def CheckFreeAlut(context,env):
             context.Message( B.bc.OKGREEN + "Linux platform detected:\n  checking for FreeAlut... " + B.bc.ENDC )
-            env['LIBS'] = 'alut'
+            env['LIBS'] = ['alut', 'openal']
             result = context.TryLink(mylib_test_source_file, '.c')
             context.Result(result)
             return result
