diff -Naur blender.orig/source/creator/CMakeLists.txt blender/source/creator/CMakeLists.txt
--- blender.orig/source/creator/CMakeLists.txt	2011-06-28 10:00:42.821785000 -0500
+++ blender/source/creator/CMakeLists.txt	2011-08-04 15:59:20.323230307 -0500
@@ -242,7 +242,7 @@
 	${CMAKE_SOURCE_DIR}/intern/guardedalloc/MEM_guardedalloc.h
 	${CMAKE_SOURCE_DIR}/source/blender/blenloader/BLO_sys_types.h
 	${CMAKE_SOURCE_DIR}/source/blender/makesdna/DNA_listBase.h
-	DESTINATION ${INCLUDE_OUTPUT_PATH})
+	DESTINATION ${INCLUDE_INSTALL_DIR})
 
 # install target
 if(UNIX AND NOT APPLE)
