diff -up blender-2.56-beta-source/extern/SConscript.ext blender-2.56-beta-source/extern/SConscript
--- blender-2.56-beta-source/extern/SConscript.ext	2010-08-29 22:52:05.000000000 +0200
+++ blender-2.56-beta-source/extern/SConscript	2011-01-16 22:15:09.530810000 +0100
@@ -2,8 +2,6 @@
 
 Import('env')
 
-SConscript(['glew/SConscript'])
-
 if env['WITH_BF_BULLET']:
     SConscript(['bullet2/src/SConscript'])
 
diff -up blender-2.56-beta-source/source/blender/editors/include/BIF_gl.h.ext blender-2.56-beta-source/source/blender/editors/include/BIF_gl.h
--- blender-2.56-beta-source/source/blender/editors/include/BIF_gl.h.ext	2010-03-06 14:43:47.000000000 +0100
+++ blender-2.56-beta-source/source/blender/editors/include/BIF_gl.h	2011-01-16 22:15:09.531810000 +0100
@@ -31,7 +31,7 @@
 #ifndef BIF_GL_H
 #define BIF_GL_H
 
-#include "../../../../extern/glew/include/GL/glew.h"
+#include <GL/glew.h>
 
 	/*
 	 * these should be phased out. cpack should be replaced in
