--- blender-2.45/source/blender/yafray/intern/export_Plugin.cpp.orig	2007-09-18 05:58:44.000000000 +0100
+++ blender-2.45/source/blender/yafray/intern/export_Plugin.cpp	2008-02-05 14:32:07.000000000 +0000
@@ -97,8 +97,8 @@
 #else
 	static char *alternative[]=
 	{
-		"/usr/local/lib/",
-		"/usr/lib/",
+		"/usr/local/lib64/",
+		"/usr/lib64/",
 		NULL
 	};
 
@@ -120,8 +120,8 @@
 #else
 	static char *alternative[]=
 	{
-		"/usr/local/lib/yafray",
-		"/usr/lib/yafray",
+		"/usr/local/lib64/yafray",
+		"/usr/lib64/yafray",
 		NULL
 	};
 
