--- blender/source/blender/python/api2_2x/CurNurb.c.gcc4	2005-05-14 16:14:26.000000000 -0400
+++ blender/source/blender/python/api2_2x/CurNurb.c	2005-05-14 16:14:42.000000000 -0400
@@ -51,7 +51,7 @@
  */
 
 
-extern PyMethodDef BPy_CurNurb_methods[];
+static PyMethodDef BPy_CurNurb_methods[];
 PyObject *CurNurb_CreatePyObject( Nurb * blen_nurb );
 static PyObject *CurNurb_setMatIndex( BPy_CurNurb * self, PyObject * args );
 static PyObject *CurNurb_getMatIndex( BPy_CurNurb * self );
